b3d/renderer/renderer.py

```diff
@@ -136,7 +136,7 @@ def _rasterize_bwd(self, saved_tensors, diffs):
         )
         dy, ddb = diffs
 
-        grads = _rasterize_bwd_custom_call(
+        _rasterize_bwd_custom_call(
```

Comment:
note to stare at this one more closely

Reply:
@nishadgothoskar can you please comment on this? -- Do we need to call `_rasterize_bwd_custom_call` here?

Reply:
(@sritchie I am not familiar with all the code in `renderer.py`, so we probably need Nishad's input here.)